File: leetcode/top-interview-questions-easy/arrays/1. remove_duplicates.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        startIndex = 0
        for i in range(len(nums)):
            if nums[i] > nums[startIndex]:
                startIndex += 1
                nums[startIndex] = nums[i]
        
        return startIndex + 1
        

File: leetcode/top-interview-questions-easy/arrays/1. remove_duplicates.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function (nums) {
  var startIndex = 0;
  for (var i = 0; i < nums.length; i++) {
    if (nums[i] > nums[startIndex]) {
      startIndex++;
      nums[startIndex] = nums[i];
    }
  }

  return startIndex + 1;
};
